eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -736,7 +738,16 @@ votingStakePoolThresholdInternal pp isElectedCommittee action =
         { pvtCommitteeNoConfidence
         , pvtCommitteeNormal
         , pvtHardForkInitiation
+        , pvtSecurityRelevant
         } = pp ^. ppPoolVotingThresholdsL
+      isSecurityRelevant (PInfo _ s) =
+        case s of
```

Comment:
You could simplify as `s == SecurityRelevant` (unless you explicitly wanted to enumerate all cases )

Reply:
I kind of prefer case matching, because that forces anyone that changes the type to revisit this piece of code.